library_generation/README.md

```diff
@@ -184,20 +188,23 @@ libraries:
       - proto_path: google/cloud/asset/v1p7beta1
 ```
 
-## An example to generate a repository using `generate_repo.py`
+## An example to generate a repository using `entry_point.py`
 
 ```bash
 # install python module (allows the `library_generation` module to be imported from anywhere)
 python -m pip install -r library_generation/requirements.in
+# install library_generation module
+python -m pip install library_generation
 # generate the repository
-python -m library_generation/generate_repo.py generate \
---generation-config-yaml=/path/to/config-file \
+python -m library_generation/entry_point.py generate \
```

Comment:
FYI, we have to remove `-m` here, this is what tripped me when I run locally, I updated it in my [PR](https://github.com/googleapis/sdk-platform-java/pull/2619)